src/components/Blog/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from "react";
+import { Row, Col, Container, Image } from "react-bootstrap";
+import "./index.css";
+
+const Blog = (props) => {
+  const storyID = props.match.params.id;
+  const _storyURL = `https://public-api.wordpress.com/wp/v2/sites/worldwithwomen30560421.wordpress.com/posts/${storyID}`;
+  console.log(storyID);
+  const [story, setStory] = React.useState({});
+
+  const getStory = async () => {
+    const data = await fetch(_storyURL);
+    const fetchedStory = await data.json();
+    setStory(fetchedStory);
+    console.log("hi");
+  };
+
+  useEffect(() => {
+    console.log("hi");
+    getStory();
+  });
```

Comment:
this should have an empty [] inside of it in order to work only once the component gets renderd

Reply:
fixed

Reply:
every time I leave the [], i receive this error:
![image](https://user-images.githubusercontent.com/67801886/98689563-1bd8a880-237d-11eb-8540-5f1d19c299b6.png)

I will try to google it more to see how can I fix it.

Reply:
@khaldoon9kh try to move the function definitions inside the useEffect

Reply:
possible solutions: useCallback() or add the function also between brakets. More explanation in the below links.  
https://stackoverflow.com/questions/55840294/how-to-fix-missing-dependency-warning-when-using-useeffect-react-hook
https://stackoverflow.com/questions/58624200/react-hook-useeffect-has-a-missing-dependency-dispatch
https://dev.to/r3wt/useeffect-missing-dependency-need-advice-4d6b